Tools/WebKitTestRunner/PlatformWin.cmake

```diff
@@ -10,7 +10,6 @@ list(APPEND WebKitTestRunner_SOURCES
 
 set(wrapper_DEFINITIONS
```

Comment:
Nit feel free to ignore but maybe turn into a one liner for consistency.

Reply:
Fixed.

Reply:
Actually looking a bit more you could probably get rid of the `wrapper_DEFINITIONS`